AppOne/AppOne/SceneDelegate.swift

```diff
@@ -16,8 +16,9 @@ class SceneDelegate: UIResponder, UIWindowSceneDelegate {
         // Use this method to optionally configure and attach the UIWindow `window` to the provided UIWindowScene `scene`.
         // If using a storyboard, the `window` property will automatically be initialized and attached to the scene.
         // This delegate does not imply the connecting scene or session are new (see `application:configurationForConnectingSceneSession` instead).
-        guard let _ = (scene as? UIWindowScene) else { return }
-        window = UIWindow()
+        guard let scene = (scene as? UIWindowScene) else { return }
+        window = UIWindow(frame: scene.coordinateSpace.bounds)
+        window?.windowScene = scene
```

Comment:
Actually there is a fitting init. `UIWindow(windowScene:)` So you can use this one and all is fine. Also i would duggest to get rid of all the empty generated `func's` we do not need atm

Reply:
updated [here]([url](https://github.com/JoeHoHH/ajayTheLead/pull/1/commits/a4e107c44a0e219e09b5d3c6165c235a278758de))